coresdk/src/coresdk/text.cpp

```diff
@@ -18,18 +18,19 @@
 
 static map<string, font> _fonts;
 
-struct _font_data
-{
-    string              name;
-    pointer_identifier  id;
-    sk_font_data        font;
-};
-
 bool has_font(string name)
 {
     return _fonts.count(name) > 0;
 }
 
+bool font_has_size(string name, int font_size)
+{
+    bool font_exists = has_font(name);
+    bool font_size_exists = _fonts[name]->_data.count(font_size) > 0;
```

Comment:
Woops, I originally had it as a one liner so it would have not worked if the font didn't exist, slipped my mind.